src/redux/index.js

```diff
@@ -0,0 +1,6 @@
+import { configureStore } from '@reduxjs/toolkit';
+import user from './user';
+
+export default configureStore({
+  reducer: user,
+});
```

Comment:
How that works with multiple reducers?

Reply:
The configureStore is similar to createStore, it accept only one reducer, but redux allow you combine reducers to, that user is a reducer, it can be combined with other ones.

Reply:
Nice, I don't read that before in the documentation.

Reply:
Here is what the interface say:
```
/**
   * A single reducer function that will be used as the root reducer, or an
   * object of slice reducers that will be passed to `combineReducers()`.
   */
  reducer: Reducer<S, A> | ReducersMapObject<S, A>
```
If you pass an object with the application reducers you'll not need to use de combineReducers() because it will make it for you.